x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -87,6 +87,7 @@ export const EndpointActionListRequestSchema = {
     pageSize: schema.maybe(
       schema.number({ defaultValue: ENDPOINT_DEFAULT_PAGE_SIZE, min: 1, max: 10000 })
     ),
+    showHostsInfo: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
This is a very UI focused param ("show*`). Maybe consider renaming it.

I see this change only for the List API. What about the Action Details?

Also - There are currently two properties that are not returned by the list API: `outputs` and `hosts`. We might (maybe) in the future also introduce an API param to expand the response with `outputs`.  because we are "expanding" the response with additional data, I would suggest maybe only having one api param named `expand` that will take in either a string or an array of strings. LIke: `expand=hosts&expand=outputs`.

Reply:
https://github.com/elastic/kibana/pull/139379#discussion_r957539132